public/src/pages/LoginPage.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+
+const LoginPage = () => {
+    return (
+        <div style={{
+            display: "flex",
+            justifyContent: "center",
+            alignItems: "center",
+            height: "100vh",
+            backgroundColor: "#0d1117",
+            color: "#c9d1d9",
+            fontFamily: "-apple-system, BlinkMacSystemFont, 'Segoe UI', Helvetica, Arial, sans-serif"
+        }}>
+            <div style={{
+                backgroundColor: "#161b22",
+                padding: "40px",
+                borderRadius: "10px",
+                boxShadow: "0 4px 14px rgba(0, 0, 0, 0.4)",
+                textAlign: "center",
+                width: "400px"
+            }}>
+                <img
+                    src="https://github.githubassets.com/images/modules/logos_page/GitHub-Mark.png"
+                    alt="GitHub Logo"
+                    style={{ width: "80px", marginBottom: "20px" }}
+                />
+                <h2>Sign in to GitHub</h2>
```

Comment:
Perhaps this should only say Sign In as the button below says `Sign in with GitHub`?